docs/about.md

```diff
@@ -103,40 +103,40 @@ We also support the [NHS Data Science Community](https://data-science-community.
     <tr><td>Giulia Mantovani</td><td>Data Science Lead</td><td>Data Linkage Hub</td><td><a href="https://github.com/GiuliaMantovani1">GiuliaMantovani1</a></td> </tr>
     <tr><td>Angeliki Antonarou</td><td>Principal Data Scientist</td><td>National SDE Data Science Team</td><td><a href="https://github.com/AngelikiA">AnelikiA</a></td> </tr>
     <tr><td>Kevin Fasusi</td><td>Principal Data Scientist</td><td>National SDE Data Science Team</td><td><a href="https://github.com/KevinFasusi">KevinFasusi</a></td> </tr>
-    <tr><td>Jonny Laidler</td><td>Senior Data Scientist</td><td>Central Data Science Team</td><td><a href="https://github.com/JonathanLaidler">JonathanLaidler</a></td> </tr>
-    <tr><td>Mia Noonan</td><td>Senior Data Scientist</td><td>Central Data Science Team</td><td><a href="https://github.com/amelianoonan1-nhs">amelianoonan1-nhs</a></td> </tr>
-    <tr><td>Sean Aller</td><td>Senior Data Scientist</td><td>Central Data Science Team</td><td><a href="https://github.com/seanaller">seanaller</a></td> </tr>
+    <tr><td>Alice Tapper</td><td>Principal Data Scientist</td><td>Central Data Science Team</td><td><a href="https://github.com/alicetapper1">alicetapper1</a></td> </tr>
+    <tr><td>Adam Hollings</td><td>Principal Data Scientist</td><td>National SDE Team</td><td><a href="https://github.com/AdamHollings">AdamHollings</a></td> </tr>
+    <tr><td>Mia Noonan</td><td>Principal Data Scientist</td><td>Central Data Science Team</td><td><a href="https://github.com/amelianoonan1-nhs">amelianoonan1-nhs</a></td> </tr>
+    <tr><td>Sean Aller</td><td>Principal Data Scientist</td><td>Central Data Science Team</td><td><a href="https://github.com/seanaller">seanaller</a></td> </tr>
+    <tr><td>Ben Wallace</td><td>Principal Data Scientist</td><td>Central Data Science Team</td><td></td> </tr>
+    <tr><td>Jonny Laidler</td><td>Principal Data Scientist</td><td>Central Data Science Team</td><td><a href="https://github.com/JonathanLaidler">JonathanLaidler</a></td> </tr>
```

Comment:
I think Jonny should be Principal Data Scientist (Section Head) as he's an 8b. He also should be moved to Data Linkage Hub rather then Central